src/pages/collection/Collection.tsx

```diff
@@ -18,18 +29,79 @@ const OptionButton = ({ icon, onClick }) => (
   </div>
 );
 
-function Collection() {
-  const { filterId } = useParams();
+const TimelineItem = ({ series }: { series: SeriesType }) => {
+  const mainPoster = useMainPoster(series);
+  const seriesType = series.AniDB?.Type === SeriesTypeEnum.TVSpecial
+    ? 'TV Special'
+    : series.AniDB?.Type;
+
+  return (
+    <Link to={`/webui/collection/series/${series.IDs.ID}`}>
+      <div className="flex gap-x-4" key={series.IDs.ID}>
+        <BackgroundImagePlaceholderDiv
+          image={mainPoster}
+          className="h-20 w-[3.4375rem] shrink-0 rounded-lg border border-panel-border drop-shadow-md"
+        />
+        <div className="flex flex-col font-semibold">
+          <div className="flex gap-y-2">
+            {moment(series.AniDB?.AirDate).year()}
+            &nbsp;|&nbsp;
+            <div className="text-panel-important">{seriesType}</div>
+          </div>
+          <div className="line-clamp-2">{series.Name}</div>
+        </div>
+      </div>
+    </Link>
+  );
+};
+
+const TimelineSidebar = ({ series }: { series: SeriesType[] }) => (
+  <div className="flex min-h-full w-[26.125rem] overflow-hidden transition-all">
```

Comment:
Shouldn't the width be 28.125rem as this width is including the gap between the two panels?

Reply:
The next line has `ml-8` which is the gap

Reply:
@ElementalCrisis Fixed